_includes/help/nav_sidenav.html

```diff
@@ -15,9 +14,9 @@
       {% else %}
         {% assign page_lang = '' %}
       {% endif %}
-      <a href="{{ subpage_slug | append: '/' | prepend: '/help/' | locale_url }}"
-        class="{% if page.url contains subpage_slug %}usa-current{% endif %}">
-        {{ site.data[page.lang]["settings"]["help_subpages"][subpage] }}
+      <a href="{{ subpage.url | prepend: '/help/' | locale_url }}"
+        class="{% if page.url contains subpage.url %}usa-current{% endif %}">
```

Comment:
I think this might have broken subpage navigation for anything other than the overview page, since `subpage.url` in the configuration is going to be `/overview/`.

Compare side navigation:

Before: https://login.gov/help/get-started/create-your-account/
After: https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mitchellhenke/redirect-from-test/help/get-started/create-your-account/

Maybe we could keep it as it was but switch up the `append: '/'` with `append: '/overview/'` ?